### DISCUSSION TITLE
while creating user using Pimcore\Model\User user not shown in "All Users" Section

### DISCUSSION THREAD

**Discussion by circlecodesolution** [pimcore/pimcore#16017] [Q&A / Support] [answered]
`<?php

use Pimcore\Model\User;

// Create a new user object.
$user = new User();

// Set the user's username.
$user->setUserName('my_username');

// Set the user's password.
$user->setPassword('my_password');

// Set the user's first name.
$user->setFirstName('My');

// Set the user's last name.
$user->setLastName('Name');

// Set the user's email address.
$user->setEmail('my_email@example.com');

// Set the user's active status to true.
$user->setActive(true);

// Save the user.
$user->save();
`

**Comment by dvesh3** [comment, marked as answer]
@circlecodesolution please set `parentId` for the user (and save) to show up in tree `$user->setParentId(0) //All Users ID`

**Reply by circlecodesolution** [reply]
thank you so much
